File: CSC101-Python/_solutions/Module05-Conditionals/debug01/letter_grade.py
# letter_grade.py
# Converts a numeric course score into a letter grade using the standard
# 90/80/70/60 cutoffs.

score = float(input("Enter the final score (0-100): "))

if score >= 90:
    grade = "A"
elif score >= 80:
    grade = "B"
elif score >= 70:
    grade = "C"
elif score >= 60:
    grade = "D"
else:
    grade = "F"

print(f"A score of {score:.1f} earns a grade of {grade}.")
if grade == "F":
    print("Please see your instructor about your options.")
else:
    print("Congratulations on passing!")

File: CSC110-Cpp/_solutions/Module04-Control-Structures-Conditionals/debug01/letter_grade.cpp
// letter_grade.cpp
// Converts a numeric score (0-100) into a letter grade using the usual
// 90/80/70/60 cutoffs.
#include <iostream>
using namespace std;

int main() {
    int score;
    char grade;

    cout << "Enter a score from 0 to 100: ";
    cin >> score;

    if (score >= 90) {
        grade = 'A';
    } else if (score >= 80) {
        grade = 'B';
    } else if (score >= 70) {
        grade = 'C';
    } else if (score >= 60) {
        grade = 'D';
    } else {
        grade = 'F';
    }

    cout << "Score: " << score << endl;
    cout << "Letter grade: " << grade << endl;
    return 0;
}
